docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,23 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned as a row for each table's column, plus a summary row for
+each table. The following table lists the returned columns and what statistics.
+they represent. Any additional statistics collected on the data source, other
+than those listed here, are not included.
 
-==========================  =============================================================
-Column                      Description
-==========================  =============================================================
-``column_name``             The name of the column (``NULL`` for the summary row)
-``data_size``               The total size in bytes of all of the values in the column
-``distinct_values_count``   The number of distinct values in the column
+==========================  ============================================================= =================================
+Column                      Description                                                   Notes
+==========================  ============================================================= =================================
+``column_name``             The name of the column                                        Returned in column rows
```

Comment:
The older text felt better to me - `NULL for the summary row` can be part of the Notes column now?